lib/eventFactory.js

```diff
@@ -194,7 +195,11 @@ function createBuilds(config) {
             // Start builds
             return Promise.all(jobsToStart.map(j =>
                 startBuild({
-                    buildConfig: Object.assign({ jobId: j.id, eventId }, eventConfig),
+                    buildConfig: Object.assign({
+                        jobId: j.id,
+                        eventId,
+                        configPipelineSha
```

Comment:
does it already have information about config pipeline's git branch and other checkout information ?

Reply:
I think it is passed here so that the `create` will pass the `configPipelineSha` to bookend.